src/FSharp.Stats/ML/Unsupervised/KNN.fs

```diff
@@ -0,0 +1,170 @@
+namespace FSharp.Stats.ML.Unsupervised
+
+
+type LabeledPoint<'a, 'l> = {
+        p     : 'a
+        label : 'l
+    }
+
+    with
+        static member create(p, l) = {
+            p     = p
+            label = l
+        }
+
+[<RequireQualifiedAccess>]
+module KNN =
+
+    open FSharp.Stats.DistanceMetrics
+
+    module Array =
```

Comment:
Again, thanks!

So indeed I added the `RequireQualifiedAccess` to prevent misuse of the `Classifier` and  `predict` functions.
Moving the  `RequireQualifiedAccess` to `Array` and `Seq` would leave the `Classifier` 'unprotected' (currently it needs to constructued with `KNN.Classifier`) and I could move it to `Array` too or some new module or simply rename it to `KnnClassifier`. 

The intention of the `Classifier` versus the `Array.predict` / `Seq.predict` versions was a more Python style version more functional style use. Moreover the `Classifier` provides convencience methods to construct the data in the required format, and to run single and multiple predictions.
Ultimately I am not sure of what the best versions are in terms of best 'user experience'.

Happy for your thoughts and feedback!